src/index.html

```diff
@@ -16,17 +16,28 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-wrapper">
```

Comment:
The `body` class name 'search-wrapper' does not seem to represent the meaning of the content. Consider using a class name that reflects the purpose or main content of the `body` element.